hkmc2/shared/src/test/mlscript/parser/Handler.mls

```diff
@@ -0,0 +1,114 @@
+
+
+:pe
+handle
+//│ ╔══[PARSE ERROR] Expected expression after 'handle' binding keyword; found end of input instead
+//│ ║  l.4: 	handle
+//│ ╙──     	      ^
+
+:pe
+handle h
+//│ ╔══[PARSE ERROR] Expected '=' keyword after 'handle' binding head; found end of input instead
+//│ ║  l.10: 	handle h
+//│ ╙──      	        ^
+
+:pe
+handle h = Eff
+//│ ╔══[PARSE ERROR] Expected 'with' keyword after 'handle' binding class name; found end of input instead
+//│ ║  l.16: 	handle h = Eff
+//│ ╙──      	              ^
+
+:pe
+handle h = Eff with
+  fun f(r) = r(0)
+  fun g(a, r) = r(1)
+//│ ╔══[PARSE ERROR] Expected 'in' keyword after type declaration block; found end of input instead
+//│ ║  l.23: 	  fun f(r) = r(0)
+//│ ╙──      	  ^
+
+:parseOnly
+handle 1 = 1 with
+  fun f(r) = r(0)
+in
+  foo(h)
+//│ Parsed:
+//│ 	Handle(IntLit(1),IntLit(1),Block(List(TermDef(Fun,App(Ident(f),Tup(List(Ident(r)))),Some(App(Ident(r),Tup(List(IntLit(0)))))))),Block(List(App(Ident(foo),Tup(List(Ident(h)))))))
+
+:e
+handle 1 = 1 with
+  fun f(r) = r(0)
+in
+  foo(h)
+//│ ╔══[ERROR] Unsupported handle binding shape
+//│ ║  l.38: 	handle 1 = 1 with
+//│ ║        	       ^^^^^^^^^^
+//│ ║  l.39: 	  fun f(r) = r(0)
+//│ ╙──      	^^^^^^^^^^^^^^^^^
+
+:elt
```

Comment:
Let's omit this flag from the test file. It generates mostly noise.

Reply:
Is there a way to run the elaborator without generating all the noise? It seems to do basic name analysis and this is something we would should test.

Reply:
The elaborator always runs even if you're not printing its result. Just remove the flag.